scripts/esbuild/build/plugins/postBuildPlugin.mjs

```diff
@@ -0,0 +1,41 @@
+import path from "node:path";
+import { copyFileSync, unlinkSync, existsSync } from "node:fs";
+
+/**
+ * Plugin to post-process build files.
+ * Equivalent to webpack's PostBuildPlugin.
+ */
+export const postBuildPlugin = (distDir, rootDir) => {
+  // it should copy the files only once. In watch mode for example it should not rerun every time
+  let completed = false;
+  return {
+    name: "post-build",
+    setup(build) {
+      build.onEnd(() => {
+        if (completed) return;
+        completed = true;
+
+        const ORT_JSEP_FILE = 'ort-wasm-simd-threaded.jsep.mjs';
+        const ORT_BUNDLE_FILE = 'ort.bundle.min.mjs';
```

Comment:
[WIP] How do we ensure this is bundled with the main library, enabling https://github.com/huggingface/transformers.js/pull/1471